query/src/main/scala/filodb/query/ResultTypes.scala

```diff
@@ -19,6 +19,11 @@ final case class QueryError(id: String, t: Throwable) extends QueryResponse with
     t.getStackTrace.map(_.toString).mkString("\n")
 }
 
+final case class RemoteQueryError(id: String,
+                                  errorResponse: RemoteErrorResponse,
+                                  statusCode: Int) extends QueryResponse {
+  override def toString: String = s"RemoteQueryError id=$id RemoteErrorResponse=$errorResponse"
```

Comment:
HTTP `StatusCode` is not needed in `toString` ?

Reply:
I forgot, will add